src/qml/TimezonePage.qml

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright (C) 2016 - Sylvia van Os <iamsylvie@openmailbox.org>
+ * Copyright (C) 2015 - Florent Revest <revestflo@gmail.com>
+ * Copyright (C) 2023 - Ed Beroset <beroset@ieee.org>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import QtQuick 2.9
+import org.asteroid.controls 1.0
+import Nemo.DBus 2.0
+
+Item {
+    id: root
+    property var pop
+    property int currentIndex: -1
+
+
+    PageHeader {
+        id: title
+        text: qsTrId("id-timezone-page")
+    }
+
+    ListModel {
+        id: continentModel
+        Component.onCompleted: {
+            var currentTZ = timedateDbus.getProperty("Timezone");
+            timedateDbus.call("ListTimezones", undefined, function(m) {
+                continentModel.clear();
+                var i = 0;
+                m.forEach(function(region) {
+                    continentModel.append({"continent": region});
+                    if (region == currentTZ) {
+                        currentIndex = i;
+                        langLV.positionViewAtIndex(i, ListView.SnapPosition)
+                    }
+                    ++i;
+                });
+            });
+        }
+    }
+
+    Spinner {
+        id: langLV
```

Comment:
continentLV?

Reply:
Cut and paste sloppiness.  I haven't spent a lot of time with this yet because I want to restructure it.  My ideal, I think, would be to have the app make up multiple levels of selection so "America/Kentucky/Monticello" would result in a top-level menu having "America" on a shorter list with other continents and UTC.  Then if you pick "America" it would show Kentucky as well as other cities/regions, and then if you select "Kentucky" it would show "Monticello" and "Louisville" as the two choices.  Because the lists are from one (UTC) to three subdivisions long, I think that the app might only show the "checkmark" to select if a terminal item is currently selected.  Comments on this proposed design are welcome!

Reply:
fixed